src/js/App/Sidenav/Navigation.js

```diff
@@ -18,6 +18,13 @@ const openshiftLinks = {
     }
 };
 
+const insightsLinks = {
+    subscriptionWatch: {
+        title: 'Subscription Watch',
+        link: `${window.location.origin}${window.insights.chrome.isBeta() ? '/beta/' : '/'}subscriptions/`
```

Comment:
It should be enough to add `./subsciptions` because we are setting the base href. You can also use `${document.baseURI}/subscription` if you want to have full URL just check if base uri contains last slash or not.